Documentation/FileModule/Index.rst

```diff
@@ -94,5 +94,6 @@ Next steps
 ==========
 
 The next chapters cover configuration and administration tasks that
-require special access privileges as described in :ref:`privileges`.
+require special access privileges as described in :ref:`Access control in the
```

Comment:
Should we change that to "Permission"? I remember a patch a week ago, where we started to rename "Access" module to "Permission" for TYPO3 13.

Reply:
Nope, Access contol and the permissions module are two different things. Therefore the chapter in TYPO3 explained still hat that title.